executor/executor.go

```diff
@@ -495,20 +495,21 @@ func init() {
 
 // HashJoinExec implements the hash join algorithm.
 type HashJoinExec struct {
-	hashTable    map[string][]*Row
-	smallHashKey []*expression.Column
-	bigHashKey   []*expression.Column
-	smallExec    Executor
-	bigExec      Executor
-	prepared     bool
-	ctx          context.Context
-	smallFilter  expression.Expression
-	bigFilter    expression.Expression
-	otherFilter  expression.Expression
-	schema       expression.Schema
-	outer        bool
-	leftSmall    bool
-	cursor       int
+	hashTable     map[string][]*Row
+	smallHashKey  []*expression.Column
+	bigHashKey    []*expression.Column
+	smallExec     Executor
+	bigExec       Executor
+	prepared      bool
+	ctx           context.Context
+	smallFilter   expression.Expression
+	bigFilter     expression.Expression
+	otherFilter   expression.Expression
+	schema        expression.Schema
+	outer         bool
+	leftSmall     bool
+	cursor        int
+	defaultValues []types.Datum
```

Comment:
It's unclear what does `defaultValues` mean in a `HashJoinExec`.
Change it to a more specific name would be better.